test/framework/src/main/java/org/elasticsearch/repositories/blobstore/AbstractBlobContainerRetriesTestCase.java

```diff
@@ -351,6 +352,7 @@ public void testReadBlobWithPrematureConnectionClose() {
             exception.getMessage().toLowerCase(Locale.ROOT),
             anyOf(
                 // closing the connection after sending the headers and some incomplete body might yield one of these:
+                containsString("premature eof"),
```

Comment:
This one is not necesesary since it was already added at the bottom of the list in #122655

Reply:
I guess that means we need backport https://github.com/elastic/elasticsearch/pull/122655

Reply:
oh, right, thanks for the pointer... i'll close this and backport https://github.com/elastic/elasticsearch/pull/122655

Reply:
I think this one should be enough, the other place didn't fail - that was just precaution